cmd/kube-apiserver/app/encryption/encryption.go

```diff
@@ -100,17 +123,18 @@ func ParseEncryptionConfiguration(f io.Reader) (map[schema.GroupResource]value.T
 	return result, nil
 }
 
-// GetPrefixTransformer constructs and returns the appropriate prefix transformers for the passed resource using its configuration
-func GetPrefixTransformers(config *ResourceConfig) ([]value.PrefixTransformer, error) {
```

Comment:
@deads2k @lavalamp Does the partial shift to k8s.io/apiserver in my previous comment seem good enough to you? Can we move ahead with that?